src/widgets/RecommendationsPaintedDoorBtn/PaintedDoorExperimentContext.jsx

```diff
@@ -68,7 +68,11 @@ export const useIsEnterpriseUser = () => {
   return enterpriseUser;
 };
 
-export const PaintedDoorExperimentContext = React.createContext();
+export const PaintedDoorExperimentContext = React.createContext({
+  experimentVariation: null,
+  isPaintedDoorWidgetBtnVariation: null,
+  experimentLoading: null,
```

Comment:
```suggestion
    experimentVariation: null,
    isPaintedDoorNavbarBtnVariation: null,
    isPaintedDoorWidgetBtnVariation: null,
    isPaintedDoorControlVariation: null,
    experimentLoading: null,
```
Missed a few properties, suggested above!

Reply:
fixed in https://github.com/openedx/frontend-app-learner-dashboard/compare/aa34eee58077f3a4230d6f6a55aa5ddaf584d6eb..0a57d8db68dc20ced8673bc5b8bb92655d7e689a